docs/source/conf.py

```diff
@@ -68,44 +68,125 @@
 # General configuration
 # ---------------------
 
+# Add any paths that contain templates here, relative to this directory.
+templates_path = config["sphinx"]["templates_path"]
+# The master toctree document.
+master_doc = config["sphinx"]["master_doc"]
+# General substitutions.
+project = config["sphinx"]["project"]
+copyright = config["sphinx"]["copyright"]
+# ghissue config
+github_project_url = config["sphinx"]["github_project_url"]
+# The suffix of source filenames.
+source_suffix = config["sphinx"]["source_suffix"]
+# Exclude these glob-style patterns when looking for source files. They are
+# relative to the source/ directory.
+exclude_patterns = config["sphinx"]["exclude_patterns"]
+# The name of the Pygments (syntax highlighting) style to use.
+pygments_style = config["sphinx"]["pygments_style"]
```

Comment:
I'm wondering if `locals().update(config["sphinx"])` would work for most (all?) of that stuff.